lib/index.js

```diff
@@ -32,22 +32,29 @@ function spawn(command, args, options, done) {
     child.on('error', onerror)
     child.on('close', onclose)
 
+    function fixUpError(error) {
```

Comment:
Amusingly, there does not seem to be a way to do the system-recognized "approved PR" without adding a comment on a line.